src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java

```diff
@@ -137,36 +141,60 @@ public static String getMockAds() {
     return getProp("mockAds");
   }
 
-  public static String getEmulator() {
-    return getProp("emulator");
+  public static Boolean getForceHttps() {
+    return getProp("forceHttps").equals("true");
+  }
+
+  public static Boolean getNewStagingUrlFormat() {
+    return getProp("newStagingUrlFormat").equals("true");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the "true" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1132)